File: code/poptipwindow/src/main/java/com/bingor/poptipwindow/impl/OnItemClickListener.java
package com.bingor.poptipwindow.impl;

import android.view.View;
import android.widget.AdapterView;

/**
 * Created by HXB on 2018/9/20.
 */
public interface OnItemClickListener<T> {
    void onItemClick(AdapterView<?> parent, View view, int position, long id, T data);

    void onItemDeleteClick(int position, T data);
}

File: code/poptipwindow/src/main/java/com/bingor/poptipwindow/impl/OnItemClickListener.kt
package com.bingor.poptipwindow.impl

import android.view.View
import android.widget.AdapterView

/**
 * Created by HXB on 2018/9/20.
 */
interface OnItemClickListener<T> {
    fun onItemClick(parent: AdapterView<*>, view: View, position: Int, id: Long, data: T)

    fun onItemDeleteClick(position: Int, data: T)
}